INSTALL.md

```diff
@@ -138,3 +138,34 @@ If `%s` is present in the value, it will be replaced with a different value for
 This allows for control of working directory cleanup after each job.
 
 The default value is `false`.
+
+### runner.max_run_time
+
+This can be used to override the default maximum time the Task Agent will run each job.
+
+The default value is 5 hours.
+
+### runner.disable_auto_update
```

Comment:
as discussed on slack, lets undocument this for now and merge the PR - then continue the conversation/decision afterwards